addons/account/models/account_tax.py

```diff
@@ -98,21 +98,21 @@ def _default_tax_group(self):
     ]
 
     @api.model
-    def default_get(self, vals):
+    def default_get(self, fields):
```

Comment:
```suggestion
    def default_get(self, fields_list):
```
better 😊, as it is defined in models.py